### DISCUSSION TITLE
Assistant api v2 support

### DISCUSSION THREAD

**Discussion by crodriguezSofteng** [microsoft/semantic-kernel#6809] [Ideas]
multi agent is nice and all, but can we get “Normal” assitant v2 support in semantic kernel without multiagent abstraction? One nice feature of assistant api (including v2) is that it saves threads automatically so you can save conversation without additional effort, but that’s been “abstracted away” in semantic kernel and you can’t recover a thread: it’s created automatically under the hood and you can’t recover it

assistant v2 with file search is very powerful and it would be nice to have sematic kernel power to use plugins. That should be your priority in my opinion. Not multiagent BEFORE fully supporting assistant api v2

**Comment by sophialagerkranspandey** [comment]
In .NET, the agent framework does support the v1 of the assistant api, we are working on supporting v2 and for python. We recommend using the OpenAI SDK for this in the interim.

**Reply by crodriguezSofteng** [reply]
you: "In .NET, the agent framework does support the v1 of the assistant api,"
me: 
no? you can't recover and interact with threads, hence "half" of what assistant api offers can't be done with SK: that you don't need to save your conversation, because assistant api does it for you.
